ICSharpCode.ILSpyX/Analyzers/Builtin/TypeExposedByAnalyzer.cs

```diff
@@ -17,15 +17,13 @@
 // DEALINGS IN THE SOFTWARE.
 
 using System.Collections.Generic;
+using System.ComponentModel.Composition;
 using System.Diagnostics;
 
+using ICSharpCode.Decompiler.TypeSystem;
```

Comment:
Ah cool, because this class was moved to ILSpyX, which does not reference WPF, we no longer have to deal with the "Accessibility.dll/namespace ambiguity" 👍